### ISSUE TITLE
version the staging directory, save a step or two?

### ISSUE THREAD

**Issue by jenlampton** [opi/boilerdrop#1]
Since Backdrop already has an importer from staging -> active, if you versioned the staging directory (like we recommend for sites on Pantheon) you could save yourself the step of needing to copy from versioned -> staging before importing, or even having a versioned directory at all!

The only potential issue with this approach is that the staging directory would end up "modified" (according to Git) after each import, as those files would be deleted.

**Comment by opi** [comment]
> The only potential issue with this approach is that the staging directory would end up "modified" (according to Git) after each import, as those files would be deleted.

This is exactly why I use the `versionned` extra folder, because `git status` is my favorite tool ! I'd like to have a clean working space.

But, as @serundeputy pointed out yesterday, backdrop now has Drush `backdrop-config-export` and `backdrop-config-import` commands, which I need to learn to improve my workflow.

But having a git modified folder continuously is a no-go for me. Maybe I have to clearly describe my workflow on the README.

**Comment by jenlampton** [comment]
Fair enough! And good idea about switching to use the drush commands, too. Thanks @serundeputy!  (I'm going to close this issue)

**Comment by herbdool** [comment]
Nitpick: "versioned" has only one "n".

We've had trouble with versioning the staging directory because of our workflow (using Beanstalk to deploy code via sftp). It can mean that some files are left in the staging directory, the diff thinks that we've removed lots of config, and then someone might accidentally import the config and delete most of the site. So we've used the more kludgy versioned folder as well.
